src/main/java/com/worktracker/service/TaskService.java

```diff
@@ -0,0 +1,16 @@
+package com.worktracker.service;
+
+import com.worktracker.model.Project;
+import com.worktracker.model.Task;
+import com.worktracker.model.dto.TaskRequestDTO;
+import com.worktracker.model.dto.TaskResponseDTO;
+
+import java.util.List;
+
+public interface TaskService {
+
+    Task createTask(TaskRequestDTO taskRequestDTO);
+
+    void updateTaskInformation(Long id, String note);
```

Comment:
Maybe we should rename this method to `updateTaskNote` as we are not updating the whole information about the task only the Note.